mesonbuild/programs.py

```diff
@@ -40,7 +40,7 @@ class ExternalProgram(mesonlib.HoldableObject):
     windows_exts = ('exe', 'msc', 'com', 'bat', 'cmd')
     for_machine = MachineChoice.BUILD
 
-    def __init__(self, name: str, command: T.Optional[T.List[str]] = None,
+    def __init__(self, name: str, command: T.Optional[T.Union[str, T.List[str]]] = None,
```

Comment:
@eli-schwartz I can move this to a separate commit if you want. If you look at the code, you can see this type has been supported for a while, but the type annotation wasn't updated.

Reply:
Yeah, support was added in commit 751f03c55767d7d9f99f13253a653bb095e798a8 -- that commit changed command from taking a `str` to also allowing `T.List[str]`, but when adding type annotations we only went with the newer supported type. Kind of ironic.

(Making it a distinct commit seems apropos.)